client/src/document/ingredients/browser-compatibility-table/index.scss

```diff
@@ -99,6 +99,9 @@
   writing-mode: vertical-rl;
   transform: rotate(180deg);
   white-space: nowrap;
+  left: calc(50% - 0.5rem);
+  position: relative;
+  padding-top: 6px;
```

Comment:
```suggestion
  padding-top: 0.5rem;
```

Reply:
rem is the root font-size, and the root has a font-size of 16px. To use equivalent spacing, it needs to be 0.375rem